website/static/js/profile.js

```diff
@@ -581,6 +602,39 @@ var ListViewModel = function(ContentModel, urls, modes) {
         }
         return true;
     });
+
+    self.institutionsEmpty = ko.computed(function() {
```

Comment:
Related to my previous comments: `ListViewModel` is an abstract class and is therefore meant to be general--it should not know about the fields of its subclasses (e.g. it should not know about "institutions" or "jobs"). Consider refactoring.